pkgs/development/python-modules/uarray/default.nix

```diff
@@ -1,33 +1,43 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , fetchpatch
 , matchpy
 , numpy
 , astunparse
 , typing-extensions
-, black
-, pytest
-, pytestcov
-, numba
-, nbval
-, python
-, isPy37
+, pytestCheckHook
+, pytestcovq
```

Comment:
```suggestion
, pytest-cov
```

?

Reply:
yeah, i just saw the failed ofborg because of my typo